probdists/Uniformdistribution.py

```diff
@@ -0,0 +1,161 @@
+            float: standard deviation of the data set
+        """
+
+        sqr_interval = (self.high - self.low) ** 2
+
+        self.stdev = math.sqrt(sqr_interval / 12)
+
+        return round(self.stdev, round_to)
+
+    def calculate_cdf(self, x, round_to=2):
+        """Cumulative distribution function calculator for the uniform distribution.
+
+                Args:
+                        x (float): point for calculating the
+                                   cumulative distribution function
+
+                Returns:
+                        float: cumulative distribution function output
+                """
+        if x < self.low:
+            self.cdf = 0
+        elif self.low<=x<=self.high:
+            self.cdf = (x - self.low)/(self.high-self.low)
+        else:
+            self.cdf = 1
+
+        return round(self.cdf, round_to)
+
+    def plot_histogram(self):
+        """Function to output a histogram of the instance variable data using
+                matplotlib pyplot library.
+
+                Args:
+                        None
+
+                Returns:
+                        None
+                """
+        plt.hist(self.data)
+        plt.title("Histogram of Data")
+        plt.xlabel("data")
+        plt.ylabel("count")
+        plt.show()
+
+    def calculate_pdf(self, x, round_to=2):
+        """Probability density function calculator for the uniform distribution.
+
+                Args:
+                        x (float): point for calculating the
+                                   probability density function
+                        round_to (int): Round the mean value. [Default value: 2 floating point]
+
+                Returns:
+                        float: probability density function output
+        """
+        self.pdf = 1/(self.high-self.low) if self.high >= x >= self.low else 0
+        return round(self.pdf, round_to)
+
+    def plot_bar_pdf(self):
+        """Function to plot the pdf of the uniform distribution
+
+        Args:
+            None
+
+        Returns:
+            list: x values for the pdf plot
+            list: y values for the pdf plot
+        """
+
+        x = []
+        y = []
+
+        # calculate the x values to visualize
+        for i in range(int(self.low)-5, int(self.high)+5):
+            x.append(i)
+            self.calculate_pdf(i)
+            y.append(self.pdf)
+
+        # make the plots
+        plt.bar(x, y)
+        plt.title('Probability Density for Uniform Distribution')
+        plt.ylabel('Probability')
+        plt.xlabel('x')
+
+        plt.show()
+
+        return x, y
+
+    def __repr__(self):
+        """Function to output the characteristics of the Uniform instance
+
+                Args:
+                        None
+
+                Returns:
+                        string: characteristics of the Uniform distribution
+
+                """
+
+        return "mean {}, standard deviation {}".format(self.mean, self.stdev)
```

Comment:
I see a (self.high - self.low) in the denominator in a lot of places. How do we handle the case when both are equal? Since it would produce division by zero, wouldn't it?